src/uwtools/scheduler.py

```diff
@@ -151,6 +152,13 @@ def batch_script(self) -> BatchScript:
 
         return BatchScript(sorted(processed))
 
+    @property
+    def submit_command(self) -> str:
+        """
+        Returns the command for running a batch script.
+        """
+        return self.command
```

Comment:
`command` is probably too generic: There are other batch-related commands, like ones to interrogate job status (which we may need at some point), one to cancel jobs (may or may not ever need), etc. So this should probably be `self.submit_command`.

Reply:
Yeah, this is what I was pondering in our meeting yesterday, but `submit_command` is better than `batch_command` or anything like that I was focusing on